src/cogito/package.d

```diff
@@ -36,6 +36,30 @@ private Result runOnFile(string file)
     return typeof(return)(visitor.source);
 }
 
+/**
+ * Map a filename to an array of files.
+ *
+ * Params:
+ *     filename = File path.
+ *
+ * Returns: Array of filenames.
+ *   If the filename points to a folder all d-files are returned,
+ *   If the filename points to a file an array with only that file is returned.
+ */
+string[] toFiles(string filename)
+{
+    import std.file : isFile, isDir, dirEntries, SpanMode;
+    if (filename.isFile)
```

Comment:
`isFile` is tricky. I just learned that it throws a `FileException` if the file doesn't exist.
```
foo: No such file or directory
```
is nice, however
```
std.file.FileException@github/cogito/tools/dmd2/linux/bin64/../../src/phobos/std/file.d(2055): foo: No such file or directory
----------------
...
??:? _Dmain [0x55891afb4f1e]
```
is not.

How about
```
try
{
   ...
}
catch (Exception exception)
{
    stderr.writefln("error: %s", expection.msg);
    exit(1);
}
```